packages/@react-spectrum/tooltip/src/TooltipTrigger.tsx

```diff
@@ -42,6 +42,8 @@ function TooltipTrigger(props: SpectrumTooltipTriggerProps) {
     trigger: triggerAction
   }, state, tooltipTriggerRef);
 
+  let arrowTotalCrossPadding = 8;
```

Comment:
why 8 for this one?
this one should be equivalent to this I think?
`border-radius: var(--spectrum-border-radius);` from
https://github.com/adobe/react-spectrum/blob/main/packages/%40adobe/spectrum-css-temp/components/tooltip/index.css#L36

Reply:
You're right, it is `border-radius * 2`. 
I noticed it's not as simple as getting `--spectrum-border-radius` because what I actually see in the compiled CSS is
`border-radius: var(--spectrum-alias-border-radius-regular, var(--spectrum-global-dimension-size-50))`. For that reason I'm updating it to use the borderRadius property value directly instead